rasa/shared/utils/schemas/domain.yml

```diff
@@ -44,8 +44,18 @@ mapping:
     type: "map"
     allowempty: True
   forms:
-    type: "any"
+    type: "map"
     required: False
+    mapping:
+      regex;([A-Za-z]+):
+        type: "map"
+        mapping:
+          required_slots:
+            type: "map"
+            required: False
```

Comment:
Should we enforce this to be `required: True` actually? 🤔 
In the docs examples I've seen, `required_slots` is always present, but not sure if giving flexibility to miss this key in forms is welcome or unwelcome.

Reply:
I put `required: False` so that you can still do 

```
forms:
  my_form:
```

Reply:
(e.g. if all slot mappings are optional)